tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -43,6 +43,8 @@
 #include "symfun.h"
 #include "ilidir.h"
 #include "fdirect.h"
+#include <string>
+#include <hash_map>
 
```

Comment:
Use of this deprecated header is causing a build failure. Please do not depend on it.

Reply:
Replaced it with unordered_map.